packages/opencrud-schema-provider/src/openCRUDSchemaProvider/openCRUDSchemaProvider.ts

```diff
@@ -7,14 +8,27 @@ import { importSchema } from 'graphql-import';
 import { DefaultParser, DatabaseType } from 'prisma-datamodel';
 import { makeExecutableSchema } from 'graphql-tools';
 import { graphqlSync, introspectionQuery } from 'graphql';
+import { sdl, datamodel } from '../test/models';
```

Comment:
To make the code clear that these are test models
```suggestion
import * as testModels from '../test/models';
```